app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/practice/CardFragment.kt

```diff
@@ -104,7 +105,13 @@ class CardFragment : AbstractFragmentWithHelp(R.string.practice_help) {
                 is MarkerSymbol -> {
                     binding.letter.visibility = View.GONE
                     binding.markerDescription.visibility = View.VISIBLE
-                    binding.markerDescription.text = inputMarkerPrintRules[it.type]
+                    binding.markerDescription.text = with(inputMarkerPrintRules[it.type]) {
```

Comment:
It is a bit strange to `with` and use `this` explicitly. `val` is better to use instead.

Reply:
Done 😃